### PR TITLE
romanian lang

### PR DESCRIPTION
**Pull request by TokRa13**
**Description:** Enabling romanian language for tedix project

**Youtrack:** https://youtrack.raccoongang.com/issue/T-238

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation in source code updated
- [ ] All related Confluence documentation is updated (including deployment documentation)
- [ ] Commits are squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: lms/envs/common.py [hunk 1]
```diff
@@ -961,7 +961,7 @@ def _add_microsite_dirs_to_default_template_engine(settings):
 
 # Locale/Internationalization
 TIME_ZONE = 'America/New_York'  # http://en.wikipedia.org/wiki/List_of_tz_zones_by_name
-LANGUAGE_CODE = 'en'  # http://www.i18nguy.com/unicode/language-identifiers.html
+LANGUAGE_CODE = 'ro'  # http://www.i18nguy.com/unicode/language-identifiers.html
```
Comment (TamoshaytisV) [inline lms/envs/common.py:964]: Do we really want to make Romanian as fallback language?

Comment (OPersian) [reply lms/envs/common.py:964]: To enable a lang,  adding `ro` in `/admin/dark_lang/darklangconfig/` to the released languages list will be enough
(then, a user will be able to choose it in account settings)

Docs https://github.com/edx/edx-platform/wiki/Internationalization-and-localization#releasing-a-language

Comment (TokRa13) [reply lms/envs/common.py:964]: That's Romanian site for romanian students. Why do you think it's a bad idea to set romaninan lang as a default one?

Comment (OPersian) [reply lms/envs/common.py:964]: It's not stated so in task requirements. Enabling a new language and setting a new default look like two different features to me.

Comment (OPersian) [reply lms/envs/common.py:964]: If I'm missing something and we really need to set `ro` as a new default, here's how we did it in other projects: 
- https://github.com/raccoongang/edx-platform/pull/1982 (and we updated LANGUAGE_LANG updated in lms.env.json; just another approach).
- similar PR: https://github.com/raccoongang/edx-platform/pull/1265

If you want to take a look:) 

Updating only the Django setting didn't seem to really work in all use cases (we had to preserve user preferences feature).

**Comment by TokRa13** [comment]
> there are no translations in [conf/locale](https://github.com/raccoongang/edx-platform/tree/tedix-hawthorn-stage/conf/locale) directory.
> 
> you need to download reviewed hawthorn revision of translations from Transifex.

Thank you. It's done now.

**Comment by a-kryachko** [comment]
LGFM. suggest to deploy this branch to STAGE environment and sent to QA. with note that Transifex has 34% for JS and 13% for Django translations. and wait for QA report.

https://www.transifex.com/open-edx/open-edx-releases/language/ro/

**Pull request merged**